url/src/test/scala/uscala/net/URLSpec.scala

```diff
@@ -69,40 +68,41 @@ class URLSpec extends Specification with ScalaCheck with TestData {
   "path" >> {
     "should decode the rawPath correctly" >> {
       "when it has a space" >> {
-        URL("http://example.com/hello%20world/").map(_.path) must beASuccessfulTry(Some("/hello world/"))
+        URL("http://example.com/hello%20world/").map(_.path) must beASuccessfulTry(Option("/hello world/"))
```

Comment:
Think both of these lines should still be `Some`?

Reply:
This is because Intellij, if I leave it as `Some`, then the line appears as a compilation error (even if it's not). So I have to either do what I did or leave as `Some` and add the type like this: `beASuccessfulTry[Option[String]](Some("/hello world/"))`.